R/post_ts.R

```diff
@@ -94,12 +92,29 @@ post_ts = function(files, on_exists=c("stop", "skip", "replace", "merge","archiv
           dir.create(post_merge_dir, showWarnings=FALSE)
           files[i] <- write_ts(data=ts_merged, site=ts_path$site_name, var=ts_path$var, src=ts_path$src, folder=post_merge_dir)
           # delete the old one in preparation for overwriting
-          item_replace_files(ts_id, files = basename(files[i]), all=FALSE)
+          item_replace_files(ts_id, files = files[i], all=FALSE)
         },
         "archive"={
-         stop("archive not yet implemented for 'post_ts'", call. = FALSE) 
+        
+         ts_fields <- item_get_fields(ts_id, 'files')
+         ts_files <- sapply(ts_fields, function(x) x$name)
+         # find which 
+         archive_indx <- which(basename(files[i]) == ts_files)
```

Comment:
for the record, i was totally confused and all of the above makes sense now. Files are pathless filenames on SB, pathed filenames on one's local computer. basename() translates to SB-speak whereas when you item_replace_files you need to give the full local path name in order to get a file name up there.